src/Processors/QueryPlan/IQueryPlanStep.h

```diff
@@ -2,6 +2,7 @@
 #include <Core/Block.h>
 #include <Core/SortDescription.h>
 #include <Processors/QueryPlan/BuildQueryPipelineSettings.h>
+#include <Processors/QueryPlan/QueryPlan.h>
```

Comment:
Let's forward-declare QueryPlan and remove this include

Reply:
Sure.